persist/fs/options.go

```diff
@@ -30,6 +31,12 @@ import (
 )
 
 const (
+	// defaultIndexSummariesPercent is the default percent to save summaries for when writing summary files
```

Comment:
This comment could be a little more clear: "defaultIndexSummariesPercent is the default percent of series for which an entry will be written into the metadata summary" or something like that. It took me awhile to figure out what this was until you explained it in person in the meeting

Reply:
For sure, will update the comment.  Thanks for the note.